format_spec.md

```diff
@@ -0,0 +1,339 @@
+
+##### `id`
+
+A unique identifier to this team, allowing you to search by Team. There is no required format for how this ID should be generated, however it MUST be unique to the Team
+
+##### `name`
+
+The human readable name of the team. Spaces are allowed.
+
+##### `uri` (optional)
+
+An optional field that can link to a webpage relating to the Team.
+
+##### `members`
+
+An array containing Author objects, who are the members of the team.
+
+**Author objects consist of the following fields:**
+
+##### `id`
+
+A unique identifier to this author. The same requirements for Team IDs should be followed here. This ID MUST be unique to the author.
+
+##### `name`
+
+The human readable name of the author. Spaces are allowed.
+
+##### `uri` (optional)
+
+An optional field that specifies a website relating to the author. It may be the author's personal website, the author's profile on a certain host, or any valid URL.
+
+---
+
+### `categories` (optional)
+
+This field MUST be stored as an array. It contains Category Objects. Those objects MUST be made up of the following fields:
+
+##### `name`
+
+The human readable name of the category. Examples include "Magic", "Technology", and "World Gen". It is NOT RECOMMENDED to use the name of a modloader as a category.
+
+##### `uri` (optional)
+
+An optional link to the category on a host, where users could see Projects within that category.
+
+##### `icon` (optional)
+
+This is an object that contains the following fields:
+
+##### `sha256` (optional)
+
+An optional SHA256 checksum of the icon specified in `uri`.
+
+##### `uri`
+
+The URI of this icon. This field is required to be present if the `icon` object is present.
+
+---
+
+### `license` (optional)
+
+This is an object that contains information about the license that a project is under. It contains the following fields:
+
+##### `spdx`
+
+The SPDX identifier of the license. [You can find a list of SPDX license names and identifiers here](https://spdx.org/licenses/).
+
+##### `uri`
+
+A URL linking to the full license.
+
+##### `modpackUsage` (optional)
+
+A field describing usage of the Project in modpacks. Valid options:
+
+- `alwaysAllow` - This Project can be used in a modpack without asking for permission.
+- `requiresPermission` - Before included in a modpack, permission must be granted from the author.
+- `neverAllow` - This Project can never be used in a modpack.
+
+---
+
+### `links` (optional)
+
+This field MUST be stored as an array. It contains a collection of links relating to the Project. Each Link Object stored inside this arrow MUST contain the following fields:
+
+##### `rel`
+
+This field MUST be stored as an array OR a String. If stored as an array, it MUST contain Strings. This field contains information about what the link is. Valid standardized values are:
+
+- `homepage` - The official homepage of a Project
+- `documentation` - Documentation of a Project.
+- `wiki` - Wiki information about a Project.
```

Comment:
Note that wiki is just a less generic name for documentation in most cases. I support using both generic and specific rel tags so that sites can change the link text for example. "Project Wiki" instead of "Project Docs" or "Project GitHub" instead of "Source code"